docs/wiki/development/building.md

```diff
@@ -47,7 +47,7 @@ cd osquery
 # Build osquery
 mkdir build; cd build
 cmake -DOSQUERY_TOOLCHAIN_SYSROOT=/usr/local/osquery-toolchain ..
-cmake --build . -j10 # where 10 is the number of parallel build jobs
+cmake --build .. -j10 # where 10 is the number of parallel build jobs
```

Comment:
I don't think this change is intentional?

Reply:
Sorry, I had that in the workspace and forgot to break it out into its own. https://github.com/osquery/osquery/pull/6947